src/main/java/oncall/domain/constants/Holiday.java

```diff
@@ -0,0 +1,32 @@
+package oncall.domain.constants;
+
+public enum Holiday {
+    NEW_YEAR(Month.JANUARY, 1),
+    INDEPENDENCE_MOVEMENT_DAY(Month.MARCH, 1),
+    CHILDREN_DAY(Month.MAY, 5),
+    MEMORIAL_DAY(Month.JUNE, 6),
+    LIBERATION_DAY(Month.AUGUST, 15),
+    NATIONAL_FOUNDATION_DAY(Month.OCTOBER, 3),
+    HANGUL_DAY(Month.OCTOBER, 9),
+    CHRISTMAS(Month.DECEMBER, 25);
+
+    private final Month month;
+    private final int day;
+
+    Holiday(Month month, int day) {
+        this.month = month;
+        this.day = day;
+    }
+
+    /**
+     * 입력한 월, 일이 법정 공휴일에 해당되는 지 확인하는 메서드
+     */
+    public static boolean isHoliday(Month month, int day) {
+        for (Holiday holiday : Holiday.values()) {
+            if (holiday.month.equals(month) && holiday.day == day) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
특정 조건을 만족하는 값이 존재하는지 확인하는 로직은 다음과 같이 작성해 줄 수 있습니다!
```suggestion
        return Arrays.stream(values())
                .anyMatch(holiday -> holiday.month.equals(month) && holiday.day == day);
```

Reply:
오 그렇군요 .. !! 자바 스트림을 잘 써보려고 했는데 역시 놓친 부분이 있었네요. 
피드백 감사드립니다!